package.json

```diff
@@ -36,7 +36,7 @@
     "clsx": "^2.1.1",
     "geist": "^1.2.1",
     "lucide-react": "^0.416.0",
-    "next": "^14",
+    "next": "^15.0.3",
```

Comment:
@pettinarip Hm, do we not need to update React as well?

https://nextjs.org/docs/app/building-your-application/upgrading/version-15

This is building just fine for me, but the docs suggest React 19 RC is required

Reply:
Ah! thats a great point! weird that I didn't have any peer dep warning 🤔 .... let me bump them